src/App.css

```diff
@@ -5,3 +5,424 @@ html { -moz-box-sizing: border-box; -webkit-box-sizing: border-box; box-sizing:
+}
+
+.user-horoscope-reading span {
+    text-transform: uppercase;
+    color: rgb(255, 248, 220);
+}
+
+.horoscope-reading h2 {
+    font-family: 'Major Mono Display', monospace;
+    font-size: 1.8rem;
+    text-align: center;
+    margin-bottom: 2%;
+}
+
+div.date-buttons {
+    width: 100%;
+}
+
+div.date-buttons button {
+    padding: 0.9% 1.5%;
+    border: none;
+    border-radius: 0.3rem;
+    color: rgb(255, 248, 220);
+    background-color: rgba(144, 181, 248, 0.295);
+    box-shadow: rgba(50, 50, 93, 0.25) 0px 30px 60px -12px inset, rgba(0, 0, 0, 0.3) 0px 18px 36px -18px inset;
+}
+
+div.date-buttons button:hover,
+div.date-buttons button:focus {
+    background-color: rgba(67, 90, 133, 0.295);
+}
+
+.yesterday {
+    margin-right: 3%;
+}
+
+.tomorrow {
+    margin-left: 3%;
+}
+
+a.arrow-up {
+    font-size: 2rem;
+    display: inline-block;
+    margin-top: 2%;
+}
+
+/* Footer */
+
+a.arrow-up:hover,
+a.arrow-up:focus {
+    color: rgb(255, 248, 220);
+}
+
+footer {
+    text-align: center;
+    height: 10vh;
+}
+
+footer p {
+    font-size: 16px;
+    padding: 1% 0;
+}
+
+footer a {
+    text-decoration: none;
+    color: white;
+}
+
+footer a:hover,
+footer a:focus {
+    color: rgb(255, 248, 220);
+}
+
+/* Media queries */
+
+@media(max-width: 1365px) {
+
+    h1 {
+        font-size: 2.8rem;
+    }
+
+    .sign-selection h2 {
+        margin-top: 10%;
+    }
+}
+
+@media(max-width: 900px) {
+
+    div.astro-buttons button {
+        height: 7.5rem;
+        width: 7.5rem;
+        padding-top: 2rem;
+    }
+
+    .sign-selection h2 {
+        margin-top: 20%;
+    }
+}
+
+@media(max-width: 815px) {
+    a.arrow-up {
+        margin-top: 5%;
+    }
+}
+
+@media(max-width: 770px) {
+
+    .sign-selection h2 {
+        margin-top: 25%;
+    }
+
+    .horoscope-reading h2 {
+        margin-top: 20%;
+    }
+
+    a.arrow-up {
+        margin-top: 25%;
+    }
+
+    footer {
+        margin-top: 5%;
+    }
+    
+}
+
+@media(max-width: 563px) {
+
+    .sign-selection h2 {
+        margin-top: 100%;
+    }
+
+    h1 {
+        font-size: 2rem;
+    }
+
+    .horoscope-reading h2 {
+        margin-top: 100%;
+    }
+
+    a.arrow-up {
+        margin-top: 110%;
+    }
+
+    footer {
+        margin-top: 5%;
+    }
+    
+}
+
+@media(max-width: 510px) {
+    .sign-selection h2 {
+        margin-top: 85%;
+        margin-bottom: 5%;
+        font-size: 1.1rem;
+    }
+
+    div.astro-buttons button {
+        height: 7rem;
+        width: 7rem;
+        font-size: 1rem;
+        padding-top: 1.7rem;
+    }
+
+    h3,
+    .reading {
+        font-size: 1.1rem;
+    }
+
+    p {
+        font-size: 1rem;
+    }
+
+}
+
+@media(max-width: 394px) {
+    h1 {
+        font-size: 1.85rem;
+    }
+
+    .sign-selection h2 {
+        margin-top: 100%;
+    }
+
+    .horoscope-reading h2 {
+        margin-top: 120%;
+    }
+
+    a.arrow-up {
+        margin-top: 125%;
+    }
+
+    footer {
+        margin-top: 10%;
+    }
+}
+
```

Comment:
Using sass could have helped eliminate some duplicate lines for setting font-families, etc., but overall look pretty dry! Good 'verbose' naming on the the class names too!